Cargo.toml

```diff
@@ -27,6 +27,7 @@ hyper-v = ["winapi", "widestring", "ntapi", "vid-sys"]
 [dependencies]
 log = "0.4.8"
 env_logger = "0.7.1"
+thiserror = "1.0.23"
```

Comment:
nit: I would (generally) not pin the patch version. The `thiserror` maintainers also suggest simply using `1.0`.